app/src/main/java/pm/gnosis/heimdall/ui/safe/add/AddSafeViewModel.kt

```diff
@@ -108,9 +111,10 @@ class AddSafeViewModel @Inject constructor(
 
     override fun addAdditionalOwner(input: String): Observable<Result<Unit>> =
             Observable.fromCallable {
-                val address = input.hexAsEthereumAddressOrNull()
+                val address = parseEthereumAddress(input)
                 SimpleLocalizedException.assert(address != null, context, R.string.invalid_ethereum_address)
-                SimpleLocalizedException.assert(deviceInfo?.let { it != address } ?: false, context, R.string.error_owner_already_added)
+                SimpleLocalizedException.assert(deviceInfo?.let { it != address }
+                        ?: false, context, R.string.error_owner_already_added)
```

Comment:
that new line is kind a weird :D

Reply:
Autoformat I think.

Reply:
Yeah, i guessed so, it's not important ;)